packages/core/src/builder.class.ts

```diff
@@ -2263,7 +2263,10 @@ export class Builder {
     url: string,
     options?: { headers: { [header: string]: number | string | string[] | undefined } }
   ) {
-    return getFetch()(url, options as SimplifiedFetchOptions).then(res => res.json());
+    return getFetch()(url, {
+      next: { revalidate: 1 },
```

Comment:
We probably wanna allow propagating `next` flags from users:

```suggestion
      next: { revalidate: 1, ...options?.next },
```